src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import * as S from "./style";
+import Arrow from '../../img/arrow.svg';
+import { PaginationType } from "../../types/pagination";
+
+const Pagination: React.FC<PaginationType> = ({
```

Comment:
Please, [avoid using React.FC](https://backstage.io/docs/architecture-decisions/adrs-adr006/). Consider keep a pattern using `function` to declare the components. If you need to receive `children`, use `PropsWithChildren` interface from React.